.github/workflows/check-manifests.yml

```diff
@@ -18,6 +18,10 @@ jobs:
     - name: Check out code
       uses: actions/checkout@v4
 
+    - name: Define missing rpm macros
```

Comment:
Would this be better in `rpmops.sh`? We use that in a few spots. Maybe something like
```bash
# Mariner macro files used during spec parsing.
MACROS=()
for macro_file in "$SPECS_DIR"/mariner-rpm-macros/macros* "$SPECS_DIR"/pyproject-rpm-macros/macros.pyproject "$SPECS_DIR"/perl/macros.perl
do
  MACROS+=("--load=$macro_file")
done

if [[ -n $(rpm --eval '%bcond test 1') ]]; then
    temp_macros="$(mktemp)"
    echo '%bcond() %[ (%{2}) ? "%{expand:%%bcond_without %{1}}" : "%{expand:%%bcond_with %{1}}" ]' > "$temp_macros"
    MACROS+=("--load=$temp_macros")
```

Reply:
Probably a good idea to add it there in addition to these two YAML files. The scripts ran with the `check-x86_64-manifests` target, however, currently use the regular `rpmspec` command, so I think a more thorough change would need to happen. I think a proper update is out of scope for this PR but I support the idea.

Reply:
to state my opinion more clearly and directly - the *problem* here is our PR runner, which IMHO is where we should put the fix/workaround. Since the problem is *not* in our toolkit code, we should not bloat/kludge our toolkit code.

Reply:
I agree that we shouldn't modify the toolkit to address issues limited to the PR runner. The story behind the `mariner_rpmspec` function from that script is just related to the issue we have with the runners. The function is a workaround to allow querying our spec files with a similar RPM configuration as what is happening inside the Mariner build chroot. But without having to build or use it (as it may not always be feasible). The functions from `rpmops.sh` are meant to be ran under Ubuntu, so there's a similar issue of an outdated RPM. At least on the latest LTS version, which is the officially supported one for the time being.

Reply:
i don't really touch the tooling at all, so you guys feel free to update it as you like, obviously; but this won't be the last new rpm macro and it certainly wouldn't be what i would do if i had to maintain the toolkit.